templates/taxi/car_list.html

```diff
@@ -1,7 +1,8 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list<a style="float: right" href="car_form.html" class="btn btn-secondary">ADD</a></h1>
```

Comment:
The 'ADD' button currently points to a static HTML file. This should be changed to point to a URL pattern name using the `{% url %}` template tag. This way, if the URL changes in the future, the link will still work. The correct usage would be `{% url 'url-pattern-name' %}`.